app/src/sharedTest/java/org/oppia/app/utility/DrawableMatcher.kt

```diff
@@ -10,11 +10,12 @@ import org.hamcrest.TypeSafeMatcher
 
 // https://github.com/dbottillo/Blog/blob/espresso_match_imageview/app/src/androidTest/java/com/danielebottillo/blog/config/DrawableMatcher.java
 /** This class mainly provides a custom matcher to test whether the drawable-image is correctly shown in ImageView. */
-class DrawableMatcher internal constructor(
+// TODO(#1655): Re-restrict access to fields in tests post-Gradle.
+class DrawableMatcher constructor(
   private val expectedId: Int
 ) : TypeSafeMatcher<View>(View::class.java) {
   companion object {
-    internal const val NONE = -1
+    const val NONE = -1
```

Comment:
This actually seems like something that shouldn’t have been internal to begin with since it’s part of this class’s API. Ditto for ANY.

Reply:
Okay

Reply:
Reminder to remove the new TODO since access doesn't actually need to be re-restricted per the comment above. :)